Security/Http/ResourceOwnerMap.php

```diff
@@ -65,6 +65,16 @@ public function setContainer(ContainerInterface $container = null)
     {
         $this->container = $container;
     }
+    
+    /**
+     * Gets service container
+     * 
+     * @return container
+     */
+    public function getContainer()
```

Comment:
This must be removed, you should inject token storage service directly into the provider instead.